tests/improver-orographic-enhancement/03-boundary_height.bats

```diff
@@ -51,11 +50,8 @@
   OUTPUT_HI_RES="20180810T1200Z-PT0006H00M-orographic_enhancement_high_resolution.nc"
```

Comment:
Just above this in line 50 there is a reference to OUTPUT_STANDARD=""20180810T1200Z-PT0006H00M-orographic_enhancement.nc" which I don't think needs to be there any more (since everything is now high res).

Reply:
Done